components/Header/header.css

```diff
@@ -0,0 +1,5 @@
+.app__card {
```

Comment:
I don't think you're using this anywhere, and you probably also shouldn't mix vanilla CSS with styled-components. 🤔

Reply:
Thank you CJ! will be removed 🫡